.github/CONTRIBUTING.md

```diff
@@ -59,6 +59,8 @@ Else GitHub issues do not add the commits to the issues. Example: `see #23`
 
  * Avoid a period/dot/`.` at the end of the commit message
  * Use the imperative, present tense "change", not "changed" nor "changes"
+ * Limit the subject line to 50 characters
+ * Use the body to explain what and why vs. how
  * One change per `<body>` line - think of it as a "list of changes", things
```

Comment:
If the body is explaining the "what and why", and there being "one change per commit", this bullet point becomes redundant.